pyproject.toml

```diff
@@ -132,6 +133,8 @@ extend-safe-fixes = [
     "UP036", # Remove unnecessary `sys.version_info` blocks
 ]
 ignore = [
+    # TODO: Link all in-code TODOs to existing issues
+    "TD003", # Missing issue link for this TODO
```

Comment:
I actually like [missing-todo-link (TD003)](https://docs.astral.sh/ruff/rules/missing-todo-link/#missing-todo-link-td003) as it ensures there's a relevant open issue (doesn't have to be on typeshed's repo) to track the required actions, offer a space for discussion, increase visibility for possible contributors, etc.

And when the issue is closed, it's easier to spot which parts of the code was affected.

However, there's currently 87 unlinked `TODO` in typeshed according to this rule.

Reply:
Not a fan of TD003 for our purposes. This would just spam our issues with small changes that not very critical. Not every instance of "dict[str, Any]  # TODO: This could be a TypedDict" needs an issue, and I don't think group issues are very useful for this either.